src/test/java/com/github/mysterix5/ProductRepoTest.java

```diff
@@ -0,0 +1,40 @@
+package com.github.mysterix5;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.*;
+
+class ProductRepoTest {
+
+    public List<Product> createDummyProducts(){
+        List<Product> products = new ArrayList<>();
```

Comment:
Die ganze Methode lässt sich auch so ein wenig kompakter schreiben:
`return List.of(new DigitalProduct("Towel"), new PhysicalProduct("Pen"));`